src/main/java/no/sikt/nva/UnisContent.java

```diff
@@ -127,7 +128,7 @@ var record = new Record();
             fileIdentifier,
             License.fromBrageLicense(getLicense()),
             getEmbargo());
-        record.setContentBundle(new ResourceContent(List.of(contentFile)));
+        record.setContentBundle(new ResourceContent(new ArrayList<>(List.of(contentFile))));
```

Comment:
assume new ArrayList<> here is to make the list mutable, maybe move it inside resourcecontent constructor?